doc/man3/SSL_shutdown.pod

```diff
@@ -80,11 +80,12 @@ It should only be done when the peer has a way to make sure all
 data has been received and doesn't wait for the close_notify alert
 message, otherwise an unexpected EOF will be reported.
 
-There are implementations that do not send the required close_notify alert.
-If there is a need to communicate with such an implementation, and it's clear
-that all data has been received, do not wait for the peer's close_notify alert.
-Waiting for the close_notify alert when the peer just closes the connection will
-result in an error being generated.
+There are implementations that do not send the required close_notify alert. If
+there is a need to communicate with such an implementation, and it's clear that
+all data has been received, do not wait for the peer's close_notify alert.
+Waiting for the close_notify alert when the peer just closes the connection
+will result in an error being generated. See more details in
+L<SSL_CTX_set_options(3)>, the option B<SSL_OP_IGNORE_UNEXPECTED_EOF>.
```

Comment:
I suggest: The error can be ignored using the B<SSL_OP_IGNORE_UNEXPECTED_EOF>. For more information see L<SSL_CTX_set_options(3)>.

Reply:
Done. Thanks!